.travis.yml

```diff
@@ -1,14 +1,12 @@
+arch:
+  - amd64
+
 language: go
-go_import_path: github.com/pkg/errors
+go_import_path: github.com/pingcap/errors
 go:
-  - 1.4.x
-  - 1.5.x
-  - 1.6.x
-  - 1.7.x
-  - 1.8.x
-  - 1.9.x
-  - 1.10.x
-  - tip
+  - 1.14.x
+  - 1.15.x
+  - 1.16.x
```

Comment:
i think we should just test on `1.13.x` (used to build TiDB currently) and the latest stable version.

Reply:
how can I specify the latest stable version? `1.16.x` or `tip` or others? I think tip is the latest devel version and cause quite a lot of time to prepare the env.

Reply:
or could you please comment with a multiline suggestion?

Reply:
@tisonkun "stable" or "1.x". https://github.com/travis-ci/gimme#asking-gimme-what-a-version-is

Reply:
```suggestion
  - 1.13.x
  - stable
```

Reply:
```suggestion
  - 1.13.x
  - stable
```